File: njpProjectAng/src/app/check-in/Checkin.ts
export class Checkin {
  id: number;
  id_leta: String;
  ime: String;
  prezime: String;
  jmbg: String;
  sediste: String;
  gate: String;
  email: String;


  constructor( id: number, id_leta: String, ime: String, prezime: String, jmbg: String, sediste: String, gate: String, email: String){
    this.id = id;
    this.id_leta = id_leta;
    this.ime = ime;
    this.prezime = prezime;
    this.jmbg = jmbg;
    this.sediste = sediste;
    this.gate = gate;
    this.email = email;
  }

}

File: RESTSpringBoot/src/main/java/org/plu/entities/CheckIn.java
package org.plu.entities;

import javax.persistence.*;

@Entity
@Table(name = "checkin")
public class CheckIn {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;

    private int id_leta;

    private String ime;

    private String prezime;

    private String jmbg;

    private String sediste;

    private String gate;

    private String email;

    public CheckIn(){}

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public int getId_leta() {
        return id_leta;
    }

    public void setId_leta(int id_leta) {
        this.id_leta = id_leta;
    }

    public String getIme() {
        return ime;
    }

    public void setIme(String ime) {
        this.ime = ime;
    }

    public String getPrezime() {
        return prezime;
    }

    public void setPrezime(String prezime) {
        this.prezime = prezime;
    }

    public String getJmbg() {
        return jmbg;
    }

    public void setJmbg(String jmbg) {
        this.jmbg = jmbg;
    }

    public String getSediste() {
        return sediste;
    }

    public void setSediste(String sediste) {
        this.sediste = sediste;
    }

    public String getGate() {
        return gate;
    }

    public void setGate(String gate) {
        this.gate = gate;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}
